lib/Basic/Subprocess.cpp

```diff
@@ -395,14 +643,17 @@ void llbuild::basic::spawnProcess(
     ProcessReleaseFn&& releaseFn,
     ProcessCompletionFn&& completionFn
 ) {
-  // Whether or not we are capturing output.
-  const bool shouldCaptureOutput = !attr.connectToConsole;
 
   // Don't use lane release feature for console workloads.
   if (attr.connectToConsole) {
     attr.controlEnabled = false;
   }
 
+#if defined(_WIN32)
+  // Control channel support is broken (thread-unsafe) on Windows.
+  attr.controlEnabled = false;
```

Comment:
WaitForMultipleObject was not configured in a way that makes it possible to use the control channel without doing a stack smash as well.